docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>This is really great and super valuable!!  And, the most important point, I learned something new regarding the <code>min_dist</code>  for ED so thank you!! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
> This is really great and super valuable!! And, the most important point, I learned something new regarding the `min_dist` for ED so thank you!!

Thanks again! This can help us to get what you wanted, making stumpy to work with both normalized and non-normalized cases. We will see how it goes :)